spec/datadog/ci/ext/fixtures/ci/appveyor.json

```diff
@@ -31,6 +37,9 @@
       "APPVEYOR_BUILD_NUMBER": "appveyor-pipeline-number",
       "APPVEYOR_REPO_BRANCH": "master",
       "APPVEYOR_REPO_COMMIT": "appveyor-repo-commit",
+      "APPVEYOR_REPO_COMMIT_AUTHOR": "appveyor-commit-author-name",
+      "APPVEYOR_REPO_COMMIT_AUTHOR_EMAIL": "appveyor-commit-author-email@datadoghq.com",
+      "APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED": "appveyor-commit-message",
       "APPVEYOR_REPO_NAME": "appveyor-repo-name",
       "APPVEYOR_REPO_PROVIDER": "github"
```

Comment:
Is something wrong with these fixtures? The first fixture is exactly the same as this second one.

Edit: I started eyeing stuff below, but since there's a lot of them, my suggestion is to maybe write a quick script that finds duplicate fixtures?

Reply:
that's a good catch! These fixtures are generated automatically with a [Makefile](https://github.com/DataDog/datadog-ci-spec/blob/main/Makefile). The base are yaml files, and of course, there are repeated test cases: https://github.com/DataDog/datadog-ci-spec/blob/main/spec_tests/appveyor.yaml#L23-L44. I've created a task to remove this and review if there are repeated fixtures elsewhere. 
Thanks!

Reply:
I opened a PR to fix this :) https://github.com/DataDog/datadog-ci-spec/pull/116